### PR TITLE
[SAIC-580] Remove admin roles from tenants

### PR DESCRIPTION
**Pull request by plomakin**
Removed admin roles assigning while creating tenants.
Added user with admin role to one of the tenants.
Fixed some of pylint issues.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1185)
devlab/tests/generate_load.py:968: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:974: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1119: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1424/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-juno/628/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Icehouse migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-icehouse/1518/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1185)
devlab/tests/generate_load.py:968: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:974: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1119: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1425/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Icehouse to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-icehouse-to-juno/428/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-juno/629/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Icehouse migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-icehouse/1519/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Icehouse to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-icehouse-to-juno/429/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1191)
devlab/tests/generate_load.py:333: [E0712(catching-non-exception), Prerequisites.create_user_tenant_roles] Catching an exception which doesn't inherit from BaseException: ks_exceptions
devlab/tests/generate_load.py:974: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:980: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1125: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1433/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Icehouse migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-icehouse/1527/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-juno/637/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Icehouse to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-icehouse-to-juno/437/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1191)
devlab/tests/generate_load.py:974: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:980: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1125: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1437/

**Comment by MirantisWorkloadMobility** [comment]
Functional test success for Grizzly to Icehouse migration

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Icehouse to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-icehouse-to-juno/440/

**Comment by MirantisWorkloadMobility** [comment]
Functional test failure for Grizzly to Juno migration
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-functional-tests-grizzly-to-juno/640/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1191)
devlab/tests/generate_load.py:974: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:980: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1125: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1443/

**Comment by MirantisWorkloadMobility** [comment]
Functional test success for Grizzly to Icehouse migration

**Comment by MirantisWorkloadMobility** [comment]
Functional test success for Grizzly to Juno migration

**Comment by MirantisWorkloadMobility** [comment]
Functional test success for Icehouse to Juno migration

**Comment by MirantisWorkloadMobility** [comment]
Ready

**Comment by roman-verchikov** [comment]
Looks good, +1

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -809,6 +814,20 @@ def modify_admin_tenant_quotas(self):
...
+            self.switch_user(self.username, self.password, self.tenant)
+            self.keystoneclient.roles.remove_user_role(
+                self.get_user_id(self.username),
+                self.get_role_id('admin'),
+                self.get_tenant_id(tenant['name']))
+            self.switch_user(self.username, self.password, self.tenant)
+            self.keystoneclient.roles.add_user_role(
+                self.get_user_id(self.username),
+                self.get_role_id('_member_'),
+                self.get_tenant_id(tenant['name']))
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: Let's remove all roles, even `_member_` to be able to test more corner cases

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1186)
devlab/tests/generate_load.py:969: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:975: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1120: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1186)
devlab/tests/generate_load.py:969: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:975: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1120: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
recheck this please

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:7: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:9: [W0403(relative-import), ] Relative import 'filtering_utils', should be 'tests.filtering_utils'
devlab/tests/generate_load.py:36: [W0621(redefined-outer-name), clean_if_exists.wrapper] Redefining name 'args' from outer scope (line 1186)
devlab/tests/generate_load.py:969: [W0120(useless-else-on-loop), CleanEnv.clean_vms.wait_until_vms_all_deleted] Else clause on loop without a break statement
devlab/tests/generate_load.py:975: [W0106(expression-not-assigned), CleanEnv.clean_vms] Expression "[vms.append(vm) for vm in self.config.vms_from_volumes]" is assigned to nothing
devlab/tests/generate_load.py:1120: [W0141(bad-builtin), CleanEnv.clean_keypairs.delete_user_keypairs] Used builtin function 'map'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: devlab/tests/generate_load.py (hunk 1 above)
Comment (plomakin) [inline devlab/tests/generate_load.py]: done

**Comment by pavloivanets** [comment]
+1 looks OK

**Comment by roman-verchikov** [comment]
+1, can be merged
